tools/diagnostic.sh

```
@@ -0,0 +1,127 @@
+    echo -e "${RED}[WARNING] docs/governance/AI-COGNITIVE-TWIN-PROTOCOL.md is MISSING.${NC}"
+    echo -e "${YELLOW}          Copy from the DSOM skeleton and fill in your project details.${NC}"
+    echo -e "${YELLOW}          See: docs/HOWTO-SETUP-ANSIBLE-BASELINE.md${NC}"
+fi
+
+# 5. ANSIBLE BASELINE CHECK
+echo -e "\n${YELLOW}Step 5: Checking Ansible Baseline...${NC}"
+ANSIBLE_OK=1
+
+if ! command -v ansible &>/dev/null; then
+    echo -e "${YELLOW}[SKIP] Ansible not installed — skipping Ansible checks (non-infra project or setup pending).${NC}"
+    ANSIBLE_OK=0
```

Shellcheck (0.11.0)

[warning] ANSIBLE_OK appears unused. Verify use (or export if used externally).

(SC2034)